features/kernel/src/main/feature/feature.xml

```diff
@@ -85,6 +125,7 @@
     </feature>
 
     <feature name="guava" version="${project.version}">
+        <bundle>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jsr305/${jsr305.version}</bundle>
```

Comment:
❓ Why was this needed for the upgrade? Are new dependencies looking for `javax.annotation` now?

Reply:
Per the release notes, Karaf moved to Annotations 1.3 
https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12311140&version=12344587
But it looks like Guava 20 is using a lower version of annotations, so we must explicitly start this bundle since it is no longer part of Karaf Standard / Enterprise profiles.